### PR TITLE
disable eclipse autosave actions

### PR DESCRIPTION
**Pull request by Bananeweizen**
Since everyone uses AS meanwhile, disable the Eclipse autosave actions
to avoid the import order getting shuffled when actually using Eclipse.

### CODE REVIEW

File: tests/.settings/org.eclipse.jdt.ui.prefs [hunk 1]
```diff
@@ -40,6 +40,8 @@ sp_cleanup.qualify_static_member_accesses_through_subtypes_with_declaring_class=
 sp_cleanup.qualify_static_member_accesses_with_declaring_class=true
 sp_cleanup.qualify_static_method_accesses_with_declaring_class=false
 sp_cleanup.remove_private_constructors=true
+sp_cleanup.remove_redundant_modifiers=false
+sp_cleanup.remove_redundant_semicolons=false
```
Comment (UniQP) [inline tests/.settings/org.eclipse.jdt.ui.prefs]: These lines looks unrelated to the commit message (and are only added in this file).

Comment (Bananeweizen) [reply tests/.settings/org.eclipse.jdt.ui.prefs]: Good catch. Fixed.

Comment (Bananeweizen) [reply tests/.settings/org.eclipse.jdt.ui.prefs]: Fixed (And what is the button "Resolve conversation" doing, which I see below this input box? I never saw that before.)

Comment (UniQP) [reply tests/.settings/org.eclipse.jdt.ui.prefs]: Resolved conversations are hidden by default.
